inst/css/BS4/pkgdown.sass

```diff
@@ -112,50 +112,6 @@ h6.dropdown-header {
   margin-top: 0;
 }
 
-.dropdown-submenu {
```

Comment:
there's no "submenu" AFAICT.

Reply:
There was a pull request that added this — can you please double check?

Reply:
Oops, re-added, and #1701 opened.